qemu/tests/cfg/ctrl_vlan.cfg

```diff
@@ -0,0 +1,20 @@
+- ctrl_vlan: 
```

Comment:
is this patch depend on https://github.com/avocado-framework/avocado-vt/pull/1751?
if so, please add it in your first conversation.

Reply:
@xiywang Thanks, update it.